src/oidc/format/json.ts

```diff
@@ -1,31 +1,11 @@
-import { resolve } from 'url';
```

Comment:
This file should not have changed, and basically should be reverted.

This file is responsible specifically for returning an OpenID formatted `userinfo` object.

It should eventually use our new user information, but it's format comes directly from a standard and changing the format means this is broken.

The purpose of files in the `format/` directory are basically rendering JSON.